reactor-core/src/test/java/reactor/core/publisher/MonoThenIgnoreTest.java

```diff
@@ -57,6 +57,14 @@ public void chained() {
 		            .expectComplete();
 	}
 
+
+	@Test
+    public void thenReturn() {
+	    StepVerifier.create(Mono.just(0).thenReturn(2))
+                    .expectNext(2)
+                    .expectComplete();
```

Comment:
same here, `verifyComplete()`. This actually made me notice that a couple of tests wrongly use `expectComplete()` without a `verify()` :(

Reply:
(I'll fix these other pre-existing bad tests on master)